flash.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+import os
+import subprocess
+import re
+
+def downloadESPtool():
+    output = subprocess.run(["sudo", "apt", "install", "esptool"], text=True)
+    if(output.returncode != 0):
```

Comment:
I think best practice doing like this:
```suggestion
try:
    subprocess.run(["sudo", "apt", "install", "esptool"], check=True)
except subprocess.CalledProcessError as err:
   raise err

```